PeakLocalisations/Localisation/Features/PeakPro.xcstrings

```diff
@@ -116,6 +116,52 @@
         }
       }
     },
+    "Feature.FamilySharing" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Gemeinsam stärker werden"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Get Stronger, Together"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Hágase Más Fuerte, Juntos"
+          }
+        }
+      }
+    },
+    "Feature.FamilySharing.Description" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Teilen Sie Peak Pro mit bis zu 6 Familienmitgliedern über die iCloud Familien-Freigabe"
```

Comment:
I've translated all the texts using the "informal" style (https://www.babbel.com/en/magazine/du-sie-formal-german), which seemed more appropriate for this kind of app.
Also, Apple does not hyphenate the "Family Sharing" (https://www.apple.com/de/family-sharing/).

```suggestion
            "value" : "Teile Peak Pro mit bis zu 6 Familienmitgliedern über die iCloud Familienfreigabe"
```

Reply:
Awesome, thank you!